<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wd_create_sql_con</class>
 <widget class="QDialog" name="wd_create_sql_con">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="lab_con_passwd">
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="edit_con_usr">
       <property name="text">
        <string>root</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="edit_con_passwd">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lab_con_ip">
       <property name="text">
        <string>目标IP：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lab_con_port">
       <property name="text">
        <string>目标端口：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lab_con_usr">
       <property name="text">
        <string>账号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edit_con_name"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lab_con_name">
       <property name="text">
        <string>连接名：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edit_con_ip">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="edit_con_port">
       <property name="text">
        <string>3306</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lab_con_db">
       <property name="text">
        <string>数据库名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edit_con_db"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,3,0">
     <item>
      <widget class="QPushButton" name="btn_connect">
       <property name="text">
        <string>测试连接</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_con_name</tabstop>
  <tabstop>edit_con_db</tabstop>
  <tabstop>edit_con_ip</tabstop>
  <tabstop>edit_con_port</tabstop>
  <tabstop>edit_con_usr</tabstop>
  <tabstop>edit_con_passwd</tabstop>
  <tabstop>btn_connect</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
